### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by estelmus**

### CODE REVIEW

File: staff/estel-mora/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#bash
+pwd: indica la carpeta actual
+ls: ensenya les carpetes dins de la carpeta actual
```
Comment (manuelbarzi) [inline staff/estel-mora/bash/README.md]: enclose the code in back ticks (follow my demo)

File: staff/estel-mora/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Some easy commands: 
+### pwd: displays the current directory path
+```sh
+$  pwd
+```
+
+### ls: lists the files and directories in the current directory
+```sh
+$ ls
+````
```
Comment (manuelbarzi) [inline staff/estel-mora/bash/README.md]: remove extra back tick

File: staff/estel-mora/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+### mkdir: create directories (folders) in the file system
+```sh
+$ mkdir "folder name"
+```
+
+### mv: move files/ directories from one location to another
+```sh
+$ mv 
+ex1: estel@MacBook-Air-de-Estel ~ % mv /path/to/source/example.txt /path/to/destination/
```
Comment (manuelbarzi) [inline staff/estel-mora/bash/README.md]: remove `ex`(amples), and put them as they are in different lines

File: staff/estel-mora/bash/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## clear:vclears the terminal screen
+```sh
+$ clear
+```
+
+## sudo: allows to execute a command as the superuser
+```sh
+$ sudo
+````
```
Comment (exogenesispr) [inline staff/estel-mora/bash/README.md:69]: remove extra back tick 👀

File: staff/estel-mora/bash/README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+## top: shows the current processes happening in the pc 
+```sh
+$ top 
+ex: CPU %, etc
+````
```
Comment (manuelbarzi) [inline staff/estel-mora/bash/README.md]: remove extra back tick

**Pull request merged**